doc/docgen.rst

```diff
@@ -14,12 +14,22 @@ Introduction
 ============
 
 This document describes the `documentation generation tools`:idx: built into
-the `Nim compiler <nimc.html>`_, which can generate HTML and JSON output
-from input .nim files and projects, as well as HTML and LaTeX from input RST
+the `Nim compiler <nimc.html>`_, which can generate HTML, Latex and JSON output
+from input ``.nim`` files and projects, as well as HTML and LaTeX from input RST
 (reStructuredText) files. The output documentation will include the module
 dependencies (`import`), any top-level documentation comments (`##`), and
 exported symbols (`*`), including procedures, types, and variables.
 
+===================   ======================   ============   ==============
+command               runs on...               input format   output format
+===================   ======================   ============   ==============
+`nim doc`:cmd:        documentation comments   ``.nim``       ``.html`` HTML
+`nim doc2tex`:cmd:    \`                       \`             ``.tex`` LaTeX
+`nim jsondoc`:cmd:    \`                       \`             ``.json`` JSON
```

Comment:
how about (in this or future PR) change this to doc2json for consistency (and making jsondoc an undocumented alias to it)

Reply:
but if you push this logic than `doc2html` shall be introduced instead of `doc`.